modules/client/src/client-common/interfaces/plugin.ts

```diff
@@ -40,16 +40,6 @@ export interface IPluginSettings {
   minDuration: number;
 }
 
-export interface ICreateProposalParams {
-  pluginAddress: string;
-  metadata: ProposalMetadata;
-  actions?: DaoAction[];
-  startDate?: Date;
-  endDate?: Date;
-  executeOnPass?: boolean;
-  creatorVote?: VoteValues;
-}
-
```

Comment:
Why removing it from here?

Reply:
because proposals are plugin specific and not something standardized in the framework of the core contracts

Reply:
Technically it makes sense, but at the same time, our own governance plugins are going to have identical traits, also on the UI side. Third party plugin developers are free to not use them, but if we want their plugins to be compatible our App, we do need to export the definition of how a proposal is represented on the UI side.

The same applies with future Finance interfaces, Membership interfaces, etc. Even if they have no used as of today.

Reply:
here we go again into the same discussions if our plugins should live in the SDK. Please tell me yes or no. 
If no I will add all this stuff back and we will have a blast if we ever wanna split it